clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -913,7 +913,8 @@ private ConsumerCoordinator buildCoordinator(Metrics metrics, List<PartitionAssi
                 retryBackoffMs,
                 defaultOffsetCommitCallback,
                 autoCommitEnabled,
-                autoCommitIntervalMs);
+                autoCommitIntervalMs,
+                null);
```

Comment:
Definitely good to have the integration tests, but the unit tests are also nice to have since they can catch problems faster and it's easier to ensure that corner cases are covered.